istioctl/cmd/analyze.go

```diff
@@ -134,7 +154,7 @@ istioctl experimental analyze -k -d false
 		"'true' to enable service discovery, 'false' to disable it. "+
 			"Defaults to true if --use-kube is set, false otherwise. "+
 			"Analyzers requiring resources made available by enabling service discovery will be skipped.")
-
+	analysisCmd.PersistentFlags().BoolVarP(&verbose, "verbose", "v", false, "enables verbose output")
```

Comment:
Nit: 99% of the time we use "enable " not "enables ".  Your choice.

Reply:
Fixed.